BE/db/index.js

```diff
@@ -279,11 +279,42 @@ async function getRecipeInfoById(recipeId) {
   }
 }
 
+// GET AVG RATING BY ID IN DB
+async function getAverageRating(recipeId){
+  try{
+    const {rows: ratings} = await client.query(`
+      SELECT rating
+      FROM reviews
+      WHERE recipeId=$1;
+    `, [recipeId]);
+
+    let ratingsSum = 0;
+    let avgRating;
+
+    if (ratings.length == 0){
+      avgRating = null;
```

Comment:
I think setting this to null might cause issues with how I'm doing the average rating stars, can we set it to 0 instead?